spec/lib/company_fetcher_bot_spec.rb

```diff
@@ -19,8 +19,8 @@ module UsXxCompaniesFetcher
 
   before do
     @dummy_connection = double('database_connection', :save_data => nil)
-    TestCompaniesFetcher.stub(:sqlite_magic_connection).and_return(@dummy_connection)
-    TestCompaniesFetcher.stub(:_http_post)
+    allow(TestCompaniesFetcher).to receive(:sqlite_magic_connection).and_return(@dummy_connection)
+    allow(TestCompaniesFetcher).to receive(:_http_post)
```

Comment:
Two reasons I think:

1.  If there is a failing spec in a file I think it might give up trying to convert them. Some of the spec failures are still outstanding.
2. Commented-out specs aren't converted

Reply:
> Some of the spec failures are still outstanding.

What do you mean? I don't see any specs failing...